data/json/professions.json

```diff
@@ -648,6 +656,46 @@
       "male": [ "boxer_shorts" ],
       "female": [ "bra", "boy_shorts" ]
     }
+  }, 
+    {
+    "type": "profession",
+    "ident": "groundskeeper",
+    "name": "Landscaper",
+    "description": "You used to mow lawns and trim hedges for the wealthy. Contract work was getting scarce even before the zombies came, but now you've got nothing left but your tools and expertise.",
+    "points": 1,
+    "skills": [ { "level": 2, "name": "fabrication" }, { "level": 2, "name": "survival" } ],
+    "items": {
+      "both": {
+        "items": [ "tank_top", "jacket_windbreaker", "hat_ball", "gloves_work", "glasses_safety", "socks", "sneakers", "pants_cargo" ],
+        "entries": [ { "item": "g_shovel" }, { "item": "scabbard", "contents-item": "machete" } ]
```

Comment:
Landscaper with a machete? I personally find it hard to believe (though I must admit I'm not an expert in New England landscaping).

Reply:
Landscapers are the reason you can buy 20 dollar machetes at Walmart in the United States. I considered a hedge trimmer except I ran into the same issue the full-sized shovel gave me; it was too large for his inventory and he dropped it.